### PR TITLE
Fix #107 - Upgrade to Storybook 3

### LINKED ISSUE #107: Migrate to Storybook 3
Refactor deprecated packages (@kadira) to @storybook.

### PR DESCRIPTION
**Pull request by shahzainb**

### CODE REVIEW

File: src/components/register/story.js [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
 import Register from './';
 
+const onChange = () => {
+  action('register on change');
+};
+
 storiesOf('Register')
   .add('User registration', () => (
-    <Register />
+    <Register handleChange={onChange} />
```
Comment (iamtmrobinson) [inline src/components/register/story.js:12]: Why did this bit have to be added?

Comment (shahzainb) [reply src/components/register/story.js:12]: To get rid of the console errors when viewing the Register component and if you typed into the input boxes.

Comment (panosvoudouris) [reply src/components/register/story.js:12]: Ok but you need the actions addon then to show the action

Comment (shahzainb) [reply src/components/register/story.js:12]: Yeah, for now it's just going to silently eat it and not show it. I want to see if I can meaningfully log the input values with the addon. Need to look into it more.

**Pull request merged**